src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -1229,6 +1229,63 @@ class spell_hun_viper_attack_speed : public SpellScriptLoader
     }
 };
 
+// -42243 - Volley (Trigger one)
+class spell_hun_volley_trigger : public SpellScriptLoader
+{
+public:
+    spell_hun_volley_trigger() : SpellScriptLoader("spell_hun_volley_trigger") {}
+
+    class spell_hun_volley_trigger_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_hun_volley_trigger_SpellScript);
+
+        void SelectTarget(std::list<WorldObject*>& targets)
+        {
+            // It's here because Volley is an AOE spell so there is no specific target to be attacked
+            // Let's select one of our targets
+            if (!targets.empty())
+            {
+                _target = *(targets.begin());
+            }
+        }
+
+        void HandleFinish()
+        {
+            if (!_target || _target->GetTypeId() != TYPEID_UNIT)
```

Comment:
```c++
_target->GetTypeId() != TYPEID_UNIT
```
What if the target is a player? Then the pet won't attack on volley.

Reply:
Yep - you are right :)